mne/preprocessing/ica.py

```diff
@@ -2261,10 +2271,8 @@ def _plot_corrmap(data, subjs, indices, ch_type, ica, label):
         try:
             nt, mt, s, mx = find_max_corrs(all_maps, target, threshold)
         except ValueError:
-            if threshold > 1:
-                logger.info("No component detected using find_outliers."
-                            " Consider using threshold='auto'")
-            return
+            raise RuntimeError("No component detected using find_outliers."
+                               " Consider using threshold='auto'")
```

Comment:
It's not really an error - it's just what happens if the algorithm doesn't pick up on anything; sometimes that's the correct response. It's like ICA not converging, which also raises warnings, not errors.